rust/core-lib/src/client.rs

```diff
@@ -147,9 +147,10 @@ impl Client {
         self.0.send_rpc_notification("alert", &json!({ "msg": msg.as_ref() }));
     }
 
-    pub fn add_status_item(&self, view_id: ViewId, key: &str, value: &str, alignment: &str) {
+    pub fn add_status_item(&self, view_id: ViewId, plugin: &str, key: &str, value: &str, alignment: &str) {
```

Comment:
I was thinking about this a bit more, and I think instead of calling this `plugin` we should call it `source`. I think it's likely that core will end up using this API, so we should be more general.